README.md

```diff
@@ -35,7 +35,7 @@ Argument | Description
 **(Optional)** `--report-format` | Valid formats:<br/>*Console*: output results to the console in plain text. **(default)**<br/>*Sarif*: output results to a file in [SARIF](https://sarifweb.azurewebsites.net) format.
 `-o` or `--output-file-path` | **(Required if `--report-format` is *Sarif*)**  File path to output SARIF results to.
 **(Optional)** `-v` or `--verbose` | Shows details about the analysis
-**(Optional)** `--run-powershell` | Also runs the PowerShell-based rules
+**(Optional)** `--run-all-rules` | Run all the rules against the templates, including non-security ones
```

Comment:
How about '--include-non-security-rules' instead?  On first glance it seems odd, and I wonder "wait, why do I have to manually specify I want to run them all?"  Having "non-security" specified up-front clarifies it.

Reply:
Also, this flag only applies to PSRule at the moment.  This might get complex if we introduce non-security checks in JSON rules as well.

Reply:
Thoughts from someone else before I refactor the name everywhere?

Reply:
@JohnathonMohr @VeraBE Alternative suggestion:

`--baseline` or `--with-baseline`

PSRule has many baselines. By default, customers get the latest security baseline for BPA or they can specify one. 

For example:

`--baseline Azure.All` or `--baseline Azure.Preview`

Reply:
I'm not sure if I like the idea of having a config file to filter the JSON rules and the baseline file for the PSRule rules, shouldn't we try to keep a consistent interface for all types of rules in the tool? Maybe by translating the config file into baseline files? Even though it sounds a bit annoying because in our config file we use rule ids and the baseline needs rule names

Reply:
I agree, we *should* keep a consistent interface for configuring all rules, regardless of where/how they're written.  That's going to require a thoughtful design though, and shouldn't block our current progress.  For now, I think ignoring the available baselines and just keeping what we have (running just security rules by default, but allowing a way to additionally run non-security rules if desired) is best; then we can figure out how to continue the integration next.